x-pack/plugins/licensing/tsconfig.json

```diff
@@ -4,7 +4,8 @@
     "outDir": "./target/types",
     "emitDeclarationOnly": true,
     "declaration": true,
-    "declarationMap": true
+    "declarationMap": true,
+    "isolatedModules": true,
```

Comment:
to enforce `export type` expressions when necessary. It prevents leaking an interface implementation.